optaplanner-integration/optaplanner-spring-boot-example/src/main/resources/application.properties

```diff
@@ -4,4 +4,9 @@
 # The solver runs for 30 seconds. To run for 5 minutes use "PT5M" (ISO 8601 format).
 optaplanner.solver.termination.spent-limit=PT30S
 
-
+# H2 database properties
+spring.datasource.url=jdbc:h2:mem:optaplanner-spring-boot-example
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
```

Comment:
I believe if we simply add h2 to the classpath, we don't need these properties at all. especially hiberante 5.3+ autodetects the driverClasName and dialect IIRC.